ruby/rspec/style_guide.md

```diff
@@ -0,0 +1,645 @@
+Do not leave line returns after `feature`, `context` or `describe`
+descriptions. It does makes the code more difficult to read and lowers the
+value of logical chunks.
+
+#### Bad Example
+
+```ruby
+describe Article do
+
+  describe '#summary' do
+
+    context 'when there is a summary' do
+
+      it 'returns the summary' do
+        # ...
+      end
+    end
+  end
+end
+```
+
+#### Good Example
+
+```ruby
+describe Article do
+  describe '#summary' do
+    context 'when there is a summary' do
+      it 'returns the summary' do
+        # ...
+      end
+    end
+  end
+end
+```
+
+### `let`, `subject`, and `before`/`after` group line returns
+
+Leave one line return after `let`, `subject`, and `before`/`after` blocks.
+
+#### Bad Example
+
+```ruby
+describe Article do
+  subject { create(:some_article) }
+  describe '#summary' do
+    # ...
+  end
+end
+```
+
+#### Good Example
+
+```ruby
+describe Article do
+  subject { create(:some_article) }
+
+  describe '#summary' do
+    # ...
+  end
+end
+```
+
+### `let`, `subject`, `before`/`after` grouping
+
+Only group `let`, `subject` blocks and separate them from `before`/`after`
+blocks. It makes the code much more readable.
+
+#### Bad Example
+
+```ruby
+describe Article do
+  subject { create(:some_article) }
+  let(:user) { create(:user) }
+  before do
+    # ...
+  end
+  after do
+    # ...
+  end
+  describe '#summary' do
+    # ...
+  end
+end
+```
+
+#### Good Example
+
+```ruby
+describe Article do
+  subject { create(:some_article) }
+  let(:user) { create(:user) }
+
+  before do
+    # ...
+  end
+
+  after do
+    # ...
+  end
+
+  describe '#summary' do
+    # ...
+  end
+end
+```
+
+### `it` block line returns
+
+Leave one line return around `it` blocks. This helps to separate the
+expectations from their conditional logic (contexts for instance).
+
+#### Bad Example
+
+```ruby
+describe '#summary' do
+  let(:item) { double('something') }
+
+  it 'returns the summary' do
+    # ...
+  end
+  it 'does something else' do
+    # ...
+  end
+  it 'does another thing' do
+    # ...
+  end
+end
+```
+
+#### Good Example
+
+```ruby
+describe '#summary' do
+  let(:item) { double('something') }
+
+  it 'returns the summary' do
+    # ...
+  end
+
+  it 'does something else' do
+    # ...
+  end
+
+  it 'does another thing' do
+    # ...
+  end
+end
+```
+
+### `before(:each)`or `before`
+
+There is no need to specify `(:each)` for `before`/`after` blocks, as it is
+the default functionality. There are almost zero cases to use `before(:all)`
+anymore, but if you do find one - just write it out like `before(:all)`
+
+#### Bad Example
+
+```ruby
+describe '#summary' do
+  before(:each) do
+    subject.summary = 'something'
+  end
+end
+```
+
+#### Good Example
+
+```ruby
+describe '#summary' do
+  before do
+    subject.summary = 'something'
+  end
+end
+```
+
+### 'should' it or 'should not' in `it` statements
+
+Do not write 'should' or 'should not' in the beginning of your `it` blocks.
```

Comment:
"at" the beginning, not "in".
"In" should be used on intangible things. In this case, we're talking about something specific (blocks), so "at" applies.